util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -261,9 +266,31 @@ endpackage
 %>\
   class ${class_name} extends ${dv_base_prefix}_reg;
     // fields
-% for f in reg.fields:
+<%
+  suffix = ""
+  start_idx = 0
+  compact_field_inst_name = ""
+  if isinstance(mr, MultiRegister):
+    if mr.compact:
```

Comment:
Didn't we agree in the meeting to also restrict this to cases when there was just one field in the register getting replicated? (I think this is `len(mr.reg.fields) == 1`) That avoids weird-looking interleavings.

Reply:
yes, but `compact` already ensure that. Let me add this condition to make it more clear

https://github.com/lowRISC/opentitan/blob/c7183dfb6873731427c7cd7cf1f4fe9af6610670/util/reggen/multi_register.py#L100-L101

Reply:
Oh! I'd completely forgotten that! :-)